### PR TITLE
Add PROMOTE_COST_BASIS for reddit-plugin-adzerk

### PR DESCRIPTION
**Pull request by zeantsoi**
:eyeglasses: @spladug @dwick 

Per discussion in Slack, this resolves the `AttributeError` in _reddit-plugin-adzerk_.

I looked into adding this as a config var (parsed as [`choices`](https://github.com/reddit/reddit/blob/master/r2/r2/lib/configparse.py#L69)), but it'd lack the [`name`](https://github.com/reddit/reddit/blob/master/r2/r2/lib/utils/utils.py#L111) property that is invoked in code.

### CODE REVIEW

File: r2/r2/models/promo.py [hunk 1]
```diff
@@ -39,7 +39,9 @@
 PROMOTE_STATUS = Enum("unpaid", "unseen", "accepted", "rejected",
                       "pending", "promoted", "finished")
 
-class PriorityLevel(object):
+PROMOTE_COST_BASIS = Enum('fixed_cpm', 'cpm', 'cpc',)
+
+
     name = ''
```
Comment (spladug) [inline r2/r2/models/promo.py]: what's going on here? it looks like all these bits that used to be in the PriorityLevel class have been orphaned?

Comment (zeantsoi) [inline r2/r2/models/promo.py]: That's me being bad at vi. Fixed.

**Comment by dwick** [comment]
:fish:

**Comment by spladug** [comment]
:fish:

**Pull request merged**